src/routes/about/+page.svelte

```diff
@@ -1,76 +1,193 @@
+        position="Treasurer"
+        name="Migdy Moya"
+      />
+      <TeamMember
+        img={placeholder}
+        position="Corresponding Secretary"
+        name="Patty Gonzalez"
+      />
+      <TeamMember
+        img={placeholder}
+        position="Recording Secretary"
+        name="Monica Scudieri"
+      />
+      <TeamMember
+        img={placeholder}
+        position="Past President"
+        name="Nancy Lyons"
+      />
+    </div>
+  </section>
+  <section class="instagram">
+    <h2 class="page-subheading">Follow us on Instagram!</h2>
+    <p>Stay up to date on all of the events around our beautiful community.</p>
+    <div class="post">
+      <div class="username">
+        <img alt="kha instagram profile" class="avatar" src={khaLogo} />
+        <p>{items[0].username}</p>
+      </div>
+      <Carousel {items} bind:activeIndex>
+        <div class="carousel-inner">
+          {#each items as item, index}
+            <CarouselItem bind:activeIndex itemIndex={index}>
+              <a href={item.permalink} target="_blank" rel="noreferrer">
+                {#if item.media_type=="VIDEO"}
+                  <img src={item.thumbnail_url} alt="instagram post" class="d-block w-100" />
+                {:else}
+                  <img src={item.media_url} alt="instagram post" class="d-block w-100" />
+                {/if}
+              </a>
+            </CarouselItem>
+          {/each}
+        </div>
+        <div class="carousel-control-left">
+          <CarouselControl direction="prev" bind:activeIndex {items} />
+        </div>
+        <div class="carousel-control-right">
+          <CarouselControl direction="next" bind:activeIndex {items}  />
+        </div>
+      </Carousel>
+      <p class="caption">{items[activeIndex].caption}</p>
+    </div>
+  </section>
 </section>
-
-<section class="container">
-  <h2 class="page-subheading">Our Team</h2>
-  <div class="team">
-    <TeamMember
-      img={placeholder}
-      position="President"
-      name="Ernie Bascuas"
-    />
-    <TeamMember
-      img={placeholder}
-      position="Vice President"
-      name="Zuriel Carbo"
-    />
-    <TeamMember
-      img={placeholder}
-      position="Treasurer"
-      name="Migdy Moya"
-    />
-    <TeamMember
-      img={placeholder}
-      position="Corresponding Secretary"
-      name="Patty Gonzalez"
-    />
-    <TeamMember
-      img={placeholder}
-      position="Recording Secretary"
-      name="Monica Scudieri"
-    />
-    <TeamMember
-      img={placeholder}
-      position="Past President"
-      name="Nancy Lyons"
-    />
-  </div>
+<section class="volunteer">
   <h2 class="page-subheading">We need your help!</h2>
   <p>There are many positions open! If you wish to volunteer, send us a message from the <a href="/contact">contact</a> page.</p>
 </section>
 
 <style> 
+  .container {
+    max-width: 75%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  
```

Comment:
I think we already have a container class in the global css, if all this is doing is adding the flex properties we should rename it to avoid confusion.

Reply:
yep i was overriding it, will change the name!